backend/src/main/java/pl/krkteam/battleships/common/dto/CoordinateDTO.java

```diff
@@ -18,6 +18,15 @@ public String toString() {
                 '}';
     }
 
+    @Override
+    public boolean equals(Object obj) {
+        if (!(obj instanceof CoordinateDTO)) {
```

Comment:
add check if obj is not a null, imlement hashCode() method

Reply:
Check if obj is not null is not required. 
Null is instance of Nothing so even if you write: 
Object obj = null;
then:
obj instanceof Object  ==> false  
Generally obj instanceof <ANYTHING> always returns false